stable/ambassador/values.yaml

```diff
@@ -72,19 +72,49 @@ service:
     # nodePort: 30443
 
   annotations:
-    getambassador.io/config: |
-      ---
-      apiVersion: ambassador/v1
-      kind: Module
-      name: ambassador
-      config:
-        service_port: 8080
```

Comment:
Even if this is the default now I'd still say it might be considered a breaking change for the chart? But not sure, suppose it would only be a problem if you for some reason are using the latest chart version with an old image version. But considering the chart also specifies an appVersion and image tag where this is already the default, it might not be 😕

Reply:
I would argue your later point that if someone was installing an earlier version where 8080 and 8443 were not default, they would need a local values file anyway. 

I suppose if you have a values file that just specifies the pre-0.60.0 image tag you are using and relying on the chart setting `service_port: 8080` by default it could be a breaking change.